src/control-plane/backend/click-stream-api.ts

```diff
@@ -313,6 +321,15 @@ export class ClickStreamApiConstruct extends Construct {
             'emr-serverless:ListApplications',
           ],
         }),
+        new iam.PolicyStatement({
+          effect: iam.Effect.ALLOW,
+          resources: [
+            `arn:${Aws.PARTITION}:iam::${Aws.ACCOUNT_ID}:role/Clickstream*`,
```

Comment:
Do you only need assume the role to execute the data-api for Redshift operations? Could we add more prefix to limit the permission?

Reply:
`role/Clickstream-DataModeling*`